Readability.js

```diff
@@ -232,6 +232,25 @@ Readability.prototype = {
     var prePath = this._uri.prePath;
     var pathBase = this._uri.pathBase;
 
+    // Use `<base>` tag's href if it exists.
+    var bases = this._doc.getElementsByTagName("base");
+    var baseHref = null;
+    if (bases.length) {
+      baseHref = bases[0].getAttribute("href");
+      if (baseHref) {
+        if (baseHref.slice(-1) === "/") {
+          prePath = baseHref.slice(0, -1);
```

Comment:
Since we've already fixed the issue in the consumer[1], let's close the pull request.

[1]: https://bugzilla.mozilla.org/show_bug.cgi?id=1173823